x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/date_range_picker.tsx

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
Two general things about this component:
- As this is an activity log date range component, it should be named something like: activity_log_date_range_picker or something like that.
- As we talked in our eng office hours, could you create a folder and move inside that folder the component, the index (for export/mport purpose) and the test files?

Reply:
Indeed 😃  Will do.

Reply:
done f551b67d661a330621fb77c435db26ec90288b81